ci/docker/ubuntu-20.04-cpp.dockerfile

```diff
@@ -75,6 +75,7 @@ RUN apt-get update -y -q && \
         libcurl4-openssl-dev \
         libgflags-dev \
         libgoogle-glog-dev \
+        libgrpc++-dev \
```

Comment:
We don't need to install `libgrpc++-dev` and `protobuf-compiler-grpc` when we use bundled gRPC and ProtoBuf.